x-pack/platform/plugins/shared/apm_sources_access/public/index.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { PluginInitializer } from '@kbn/core/public';
+import {
+  ApmSourceAccessPlugin,
+  type ApmSourceAccessPluginSetup,
+  type ApmSourceAccessPluginStart,
+} from './plugin';
+
+export const plugin: PluginInitializer<
+  ApmSourceAccessPluginSetup,
+  ApmSourceAccessPluginStart
+> = () => new ApmSourceAccessPlugin();
+
+export type { ApmSourceAccessPluginStart, ApmSourceAccessPluginSetup };
+
+export type { APMIndices } from '../common/config_schema';
+
+export { useSourcesAPIFetcher } from './hooks';
+export { callSourcesAPI } from './api';
```

Comment:
Could we expose these as services? 

instead of exposing `useSourcesAPIFetcher`, there could be service called `getApmIndexSettings`

Same for `callSourcesAPI`, which is used to call `POST /internal/apm-sources/settings/apm-indices/save`. Instead, there could be a service like `saveApmIndicesSettings`

Consumers of these services don't need to know anything about the endpoint being called.